arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-morocco.dts

```diff
@@ -619,6 +620,24 @@
 		assigned-address = <0x3c>;
```

Comment:
Looks like commit message is just a copy of previous commit , Not specific to this commit?